initialization/initialization_test.go

```diff
@@ -443,6 +443,10 @@ func Test_SessionNumLabelsWrittenChan_Racefree(t *testing.T) {
 
 func assertNumLabelsWrittenChan(init *Initializer, t *testing.T) func() error {
 	return func() error {
+		// hack to avoid receiving a nil channel from SessionNumLabelsWrittenChan()
+		// TODO (mafa): for a proper fix see https://github.com/spacemeshos/post/issues/78
+		time.Sleep(10 * time.Millisecond)
+
```

Comment:
Kudos for the TODO with a link ❤️.

Perhaps you could use `require.Eventually(...)` with a longer timeout for receiving the channel instead? It would be less flaky than this.